src/assets/styles/other/_tn-styles.scss

```diff
@@ -927,8 +927,8 @@ $primary-dark: darken(map-get($md-primary, 500), 8%);
       @include variable(color, --primary, $primary);
     }
 
-    &.fn-theme-grey2 {
-      @include variable(color, --grey2, $grey2);
+    &.fn-theme-grey {
+      @include variable(color, --grey, --grey);
```

Comment:
The last parameter is supposed to be a fallback value. You should put a proper css color value there